src/__swaps__/screens/Swap/hooks/useSearchCurrencyLists.ts

```diff
@@ -75,7 +76,7 @@ export function useSearchCurrencyLists() {
   // static search data
   const { data: verifiedAssets, isLoading: verifiedAssetsLoading } = useTokenSearch(
     { list: 'verifiedAssets' },
-    { staleTime: 60 * 60 * 1000, cacheTime: 24 * 60 * 60 * 1000 }
+    { select: filterNonTokenIconAssets, staleTime: 60 * 60 * 1000, cacheTime: 24 * 60 * 60 * 1000 }
```

Comment:
two questions:
1. should we apply this filter to the other search queries (target verified/unverified)?
2. should this filter be at the response parsing level instead of selector level? assuming we will use the verifiedAssets elsewhere in the app, maybe we'd want the same standard of data completeness (must have icon url) everywhere in the app

Reply:
1. No. We only want this requirement for curated assets. Verified / Unverified assets should be able to not have this in case a user searches for a specific token.
2. Again related to #1, but I think we only want this for the static curated assets list